### ISSUE TITLE
Mobilenet v3 pretrained checkpoint

### ISSUE THREAD

**Issue by hussam789** [EunhyeokPark/PROFIT#4]
Hey,
I'm interested in training mobilenet v3 for quantization, can you please upload mobilenet_v3_pad.pth ?

Thanks

**Comment by EunhyeokPark** [comment]
I think you can get a similar result from the checkpoint of the original Mobilenet-v3 without negative padding.

**Comment by pigen01** [comment]
No, I can't. I use the official [mobilenet-v3-large weight](https://download.pytorch.org/models/mobilenet_v3_large-8738ca79.pth) for quantization, but get 0@pred1.
